src/Compilers/Core/Portable/Optional.cs

```diff
@@ -56,5 +56,15 @@ public T Value
         {
             return new Optional<T>(value);
         }
+
+        /// <summary>
+        /// Returns a string representation of the optional value.
+        /// </summary>
+        public override string ToString()
+        {
+            return _hasValue
+                ? _value.ToString()
```

Comment:
Hmm actually why do we use `"null"` for no value? Isn't `"default"` more appropriate given T can be `struct` or `class`